docs/content/posts/2023-10-10-release-1.18.0.md

```diff
@@ -0,0 +1,554 @@
+- [FLIP-296: Extend watermark-related features for SQL](https://cwiki.apache.org/confluence/display/FLINK/FLIP-296%3A+Extend+watermark-related+features+for+SQL)
+
+## Batch Execution Improvements
+
+### Hybrid Shuffle supports Remote Storage
+
+Hybrid Shuffle supports storing the shuffle data in remote storage. The remote storage path can be configured with the 
+option `taskmanager.network.hybrid-shuffle.remote.path`. Hybrid Shuffle uses less network memory than before by 
+decoupling the memory usage from the number of parallelisms, improving the stability and ease of use. 
+
+**More Information**
+* [Documentation](https://nightlies.apache.org/flink/flink-docs-stable/docs/ops/batch/batch\_shuffle/#hybrid-shuffle)
+* [FLIP-301: Hybrid Shuffle supports Remote Storage](https://cwiki.apache.org/confluence/display/FLINK/FLIP-301%3A+Hybrid+Shuffle+supports+Remote+Storage)
+
+### Performance Improvements & TPC-DS Benchmark
+
+In previous releases, the community worked extensively to improve Flink's batch processing performance, which has led to 
+significant improvements. In this release cycle, community contributors continued to put significant effort into 
+further improving Flink's batch performance.
+
+#### Runtime Filter for Flink SQL
+
+Runtime filter is a common method for optimizing Join performance. It is designed to dynamically generate filter 
+conditions for certain Join queries at runtime to reduce the amount of scanned or shuffled data, avoid unnecessary I/O 
+and network transmission, and speed up the query. We introduced runtime filters in Flink 1.18, and verified its 
+effectiveness through the TPC-DS benchmark, and observed up to 3x speedup for some queries by enabling this feature.
+
+#### Operator Fusion Codegen for Flink SQL
+
+Operator Fusion Codegen improves the execution performance of a query by fusing an operator DAG into a single optimized 
+operator that eliminates virtual function calls, leverages CPU registers for intermediate data and reduces the 
+instruction cache miss. As a general technical optimization, we verified its effectiveness through TPC-DS, and 
+only some batch operators (Calc, HashAgg, and HashJoin) completed fusion codegen support in version 1.18, getting 
+significant performance gains on some query.
+
+Note that both features are considered experimental and disabled by default in Flink 1.18. 
+They can be enabled by using `table.optimizer.runtime-filter.enabled` and `able.exec.operator-fusion-codegen.enabled` 
+respectively.
+
+Since Flink 1.16, the Apache Flink Community has been continuously tracking the performance of its batch engine via the 
+TPC-DS benchmarking framework. After significant improvements in Flink 1.17 (dynamic join-reordering, 
+dynamic local aggregations), the two improvements described in the previous sections lead to 12% performance improvement
+compared to Flink 1.17 , a 35% performance improvement compared to Flink 1.16 on a 10T dataset for partitioned tables.
```

Comment:
I think the numbers 12% and 35% are the reduction of TPC-DS execution time. So actually the performance improvements numbers should be 14% and 54%.